bentoml/yatai/yatai_service_impl.py

```diff
@@ -147,6 +147,24 @@ def DeleteDeployment(self, request, context=None):
                     self.deployment_store.delete(
                         request.deployment_name, request.namespace
                     )
+                elif response.status.status_code == status_pb2.Status.NOT_FOUND:
+                    self.deployment_store.delete(
```

Comment:
since we provided this flag, when cloud resource not found, let's just return an internal error status, with message "cloud resource not found, it may have been deleted manually. Try delete deployment with --force option to ignore this error." - notifying end user something is wrong and can't be handled by bentoml.